platform/qt/resources/common.qrc

```diff
@@ -1,6 +1,6 @@
 <RCC>
     <qresource prefix="/">
-        <file alias="icon.png">../../../common/icon.png</file>
+        <file alias="icon.png">../../../common/mb-icon-blue-square.png</file>
```

Comment:
But answering, let's keep it `png` otherwise we add the dependency on `QSvg` to run the test app.